src/Pelagos/Bundle/AppBundle/Controller/UI/DatasetReviewController.php

```diff
@@ -379,8 +379,6 @@ private function createNewDatasetSubmission(DatasetSubmission $datasetSubmission
         // The latest submission is complete, so create new one based on it.
         $datasetSubmission = new DatasetSubmission($datasetSubmission);
         $reviewedBy = $this->getUser()->getPerson();
-        $reviewStartTimeStamp = new \DateTime('now', new \DateTimeZone('UTC'));
-        $datasetSubmissionReview = new DatasetSubmissionReview($datasetSubmission, $reviewedBy, $reviewStartTimeStamp);
```

Comment:
Could have left the datasetSubmission in, and then called the setting of the review on it with this.